src/plugins/intel_cpu/src/nodes/lora.cpp

```diff
@@ -47,8 +47,14 @@ void LoRA::selectOptimalPrimitiveDescriptor() {
     std::vector<PortConfig> inConfs;
     std::vector<Input::InputConfig> graphInputConfig;
 
-    for (size_t i = 0; i < getParentEdges().size(); i++) {
-        auto desc = getParentOutputMemDesc(getParentEdgeAt(i));
+    auto mainInputDesc = getParentOutputMemDesc(getParentEdgeAt(0));
+    auto mainInputPrc = mainInputDesc->getPrecision(); // we have to align precision across all the inputs
```

Comment:
looks like very strict limitation. What is the tech reason and are there plans to resolve it?